### ISSUE TITLE
Is this unmaintained? 

### ISSUE THREAD

**Issue by AndreJoaquim** [spencer516/ember-cli-deploy-new-relic-sourcemap#6]
@spencer516 This plugin has several forks already, and is not under active development since 2 years ago... Is this plugin maintained?

At Uniplaces we're actively using it. Maybe did you consider giving ownership? This is not even published in npm 😅 You could add this add-on to https://github.com/adopted-ember-addons where adopted ember add-ons live happily!

**Comment by spencer516** [comment]
Hey @AndreJoaquim — yes, this is definitely not maintained anymore (at least by me!) I think I built this, then discovered someone else had built and [published it](https://www.npmjs.com/package/ember-cli-deploy-newrelic-sourcemaps). 

I'm happy to transfer ownership to whomever; adopted ember addons seems like a reasonable place. Does that work for you?  Or, I can also transfer ownership directly to you if you'd like.

**Comment by AndreJoaquim** [comment]
Hey @spencer516! Thanks for your quick reply! We've actually tried to use [that other plugin](https://www.npmjs.com/package/ember-cli-deploy-newrelic-sourcemaps) but for some reason I don't quite remember we've ditched it to use yours instead.

adopted-ember-addons seems a very good place for me, I think it'd be better than transferring the ownership to me :)

**Comment by spencer516** [comment]
Fair enough! Glad this is working for you. I'll start the conversation to get this transferred to `ember-adopted-addons`. 

In the meantime, I'll merge your PR — it looks good to me!

**Comment by AndreJoaquim** [comment]
Thanks! Just a final question: is there any reason for you to not have published this to `npm`?

**Comment by spencer516** [comment]
None that I can remember — I think I was actually trying to merge this with the other plugin so there weren't two similarly-named addons, but I never heard back from those folks. (and then I think we stopped using New Relic, so I stopped updating this).

**Comment by AndreJoaquim** [comment]
That's fine to me, I was just wondering (because the `README.md`'s install instructions don't execute successfully)! We'll just keep using `git` to install the plugin :)
